### PR TITLE
Use less chromedp on admin controller tests

### PR DESCRIPTION
**Pull request by sethvargo**
**Release Note**

```release-note
Use less chromedp on admin controller tests
```

### CODE REVIEW

File: pkg/controller/admin/mobile_apps_test.go [hunk 1]
```diff
@@ -103,73 +71,15 @@ func TestAdminMobileApps(t *testing.T) {
...
-
-			chromedp.Navigate(`http://`+harness.Server.Addr()+`/admin/mobile-apps`),
-			chromedp.WaitVisible(`body#admin-mobileapps-index`, chromedp.ByQuery),
-
-			chromedp.SetValue(`input#search`, "test mobile app", chromedp.ByQuery),
-			chromedp.Submit(`form#search-form`, chromedp.ByQuery),
-			chromedp.WaitVisible(`table#results-table tr`, chromedp.ByQuery),
-
-			chromedp.SetValue(`input#search`, "notexists", chromedp.ByQuery),
-			chromedp.Submit(`form#search-form`, chromedp.ByQuery),
```
Comment (whaught) [inline pkg/controller/admin/mobile_apps_test.go:168]: Wasn't this exercising more before - it actually runs a search.

File: pkg/controller/admin/events_test.go [hunk 2]
```diff
@@ -162,88 +109,16 @@ func TestAdminEvents(t *testing.T) {
...
-			browser.SetCookie(cookie),
-
-			// Visit /admin
-			chromedp.Navigate(`http://`+harness.Server.Addr()+`/admin/events`),
-
-			// Wait for render.
-			chromedp.WaitVisible(`body#admin-events-index`, chromedp.ByQuery),
-
-			// Search from and hour before to and hour after our event
-			chromedp.SetValue(`#from`, eventTime.Add(-time.Hour).Format(rfc3339PartialLocal), chromedp.ByQuery),
```
Comment (whaught) [inline pkg/controller/admin/events_test.go:231]: Same here. This exercised the search-by-time filter. Are we okay dropping these assertions?

File: pkg/controller/admin/mobile_apps_test.go (hunk 1 above)
Comment (sethvargo) [reply pkg/controller/admin/mobile_apps_test.go:168]: Fixed

File: pkg/controller/admin/events_test.go (hunk 2 above)
Comment (sethvargo) [reply pkg/controller/admin/events_test.go:231]: Fixed

**Review by whaught — APPROVED** [review]

**Pull request merged**